test/cpp/api/integration.cpp

```diff
@@ -336,7 +336,7 @@ TEST_CASE("integration/mnist", "[cuda]") {
   auto conv1 = model->add(Conv2d(1, 10, 5), "conv1");
   auto conv2 = model->add(Conv2d(10, 20, 5), "conv2");
   auto drop = Dropout(0.3);
-  auto drop2d = Dropout2d(0.3);
+  auto drop2d = Dropout(0.3);
```

Comment:
What happened here? Dropout isn't the same with Dropout2d.

Reply:
Oh, I somehow assumed we were figuring out whether to apply 1-d or 2-d dropout inside the `at::dropout` call because I didn't see a `at::dropout2d` Function. Looking at `functional.py` I now see this function is called `feature_dropout`. I'll fix this PR then and also add Dropout3d while I'm at it.

Reply:
Maybe not in this PR, but we should make feature dropout 1d to just call dropout in aten.